features/generate_todo_list.feature

```diff
@@ -0,0 +1,48 @@
+Feature:
+
+  Write a Reek configuration as a kind of todo list that will prevent Reek
+  from reporting smells on the current code.
+  This can then be worked on later on.
+  The main goal here would be to ease the Reek adoption by allowing developers to:
+    - introduce Reek right away (e.g. for CI)
+    - exclude the "old" smells from getting reported
+    - fix them step by step
+    - get rid of the todo file
+
+  Background:
+    Given a directory 'lib' with one clean file 'clean.rb' and one dirty file 'dirty.rb'
+
+  Scenario: Generate a proper todo file
+    When I run reek --todo lib
+    Then it succeeds
+    And the file 'todo.yml' exists
+    And the todo file looks like this:
+      """
+      ---
+      IrresponsibleModule:
+        exclude:
+        - Dirty
+      UncommunicativeMethodName:
+        exclude:
+        - Dirty#a
+        - Dirty#b
```

Comment:
I love the specificity of the generated todo; very easy to tackle the issues one-by-one. :)